calico-vpp-agent/cni/network_vpp.go

```diff
@@ -213,13 +213,15 @@ func (s *Server) AddVppInterface(podSpec *storage.LocalPodSpec, doHostSideConf b
 
 	// Create new tun
 	tun := &types.TapV2{
+		GenericVppInterface: types.GenericVppInterface{
+			NumRxQueues:   config.TapNumRxQueues,
```

Comment:
This (and other struct definitions through the PR) doesn't look gofmt'ed properly?

Reply:
Oh right, my bad, forgot to re-run the script
Should be right now !